nengo/learning_rules.py

```diff
@@ -197,6 +201,42 @@ def _argreprs(self):
         return args
 
 
+class Voja(LearningRuleType):
+    """Vector Oja's learning rule.
+
+    Modifies an ensemble's encoders to be selective to its inputs.
+
+    A connection to the learning rule will provide a scalar weight for the
+    learning rate (minus 1). For instance, 0 is normal learning, -1 is no
+    learning, and less than -1 causes anti-learning or "forgetting".
+
+    Parameters
+    ----------
+    learning_rate : float, optional
```

Comment:
Looking elsewhere in the repo, I can't find a consistent rule for the ordering. Sometimes it's neither alphabetical nor order of appearance. Perhaps we should pick a convention and then do a giant sweep over in another PR.